content/en/docs/ops/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Upgrade with Helm
+description: How to upgrade Ambient Mesh with Helm.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+Follow this guide to upgrade and configure ambient mesh using
+[Helm](https://helm.sh/docs/).  This guide assumes you have already performed an ambient mesh
+[installation with Helm](/docs/ops/ambient/install/helm-installation/) for a previous minor or patch version of Istio.
+
+{{< warning >}}
```

Comment:
Replace with ambient-alpha-warning template.

Reply:
Found another spot that should use this too, will roll that up as well.